### DISCUSSION TITLE
contract_terms/Replacing: "Terms and conditions of the lender apply."

### DISCUSSION THREAD

**Discussion by MatKDA** [leihs/leihs#1268] [Q&A / Community Support] [answered]
Hello,

#### Expected behaviour

We would like to replace the phrase “Terms and conditions of the lender apply.” (in the contract) with the complete terms and conditions. In the “Settings” menu (“Admin” section), there is the promising “contract_terms” field.

#### Actual behaviour

Editing this field (contract_terms) doesn’t seem to have any (visible?) effect.

#### Steps to reproduce

Log-in as admin, edit the "contract_terms" field (Settings).

#### Specifications
- Leihs version: 5.3.1
- Browser version: Firefox 78.10.0esr (64-Bit)

Of course, I can change the phrase in the .po file, however, we are not sure how to insert line breaks/links here.

Now we are wondering: Is it even possible to replace the sentence?

Thanks in advance,
Matthias

**Comment by eins78** [comment, marked as answer]
* the setting `contract_terms` had no effect in Leihs v5 and was removed in Leihs v6

* technically you can edit any translations strings before deploying, but its unsupported. a few hints: 1. make sure the deploy scripts dont have `git submodule update…` commands in them that would revertt your changes; 2. only edit in `legacy/public/assets` repo, the sources will not be used/recompiled during deploy; 3.search for `\n` to find linebreak examples.


I would also be interested what kind of content you would like to add there. Would it be lots of text? Would it be sufficient if you configure a Link/URL (external to Leihs, like a page or PDF on your website) in the settings that would be included after this sentence?
